File: 0x12-javascript-warm_up/7-multi_c.js
#!/usr/bin/node
// prints message x times
let x = parseInt(process.argv[2], 10);
if (isNaN(x)) {
  console.log('Missing number of occurrences');
}
if (x > 0) {
  for (; x > 0; x--) {
    console.log('C is fun');
  }
}

File: 0x12-javascript-warm_up/Rusty/7-multi_c.rs
// prints message x times
use std::env;

fn main() {
	let args: Vec<String> = env::args().collect();

	if args.len() > 1 {
		let x = match args[1].parse() {
			Ok(n) => n,
			Err(_) => { println!("Missing number of occurrences"); 0 },
		};
		if x > 0 {
			for i in 0..x {
				println!("Rust is Awesome");
			}
		}
	}
}
